From: Simon McVittie <smcv@debian.org>
Date: Sat, 8 Mar 2025 10:05:20 +0000
Subject: sysusers.d: Set user's home directory to /nonexistent

sysusers.d(5) defaults to setting the root directory as home directory,
but the canonical placeholder home directory in Debian is /nonexistent.

Forwarded: not-needed
---
 bus/sysusers.d/dbus.conf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bus/sysusers.d/dbus.conf.in b/bus/sysusers.d/dbus.conf.in
index fb35702..7f7b2af 100644
--- a/bus/sysusers.d/dbus.conf.in
+++ b/bus/sysusers.d/dbus.conf.in
@@ -2,4 +2,4 @@
 # at boot on systemd-based systems that ship with an unpopulated
 # /etc. See sysusers.d(5) for details.
 
-u @DBUS_USER@ - "System Message Bus"
+u @DBUS_USER@ - "System Message Bus" /nonexistent
